app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  include VotesHelper
   EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
just fyi (because I just learned this myself too), Ruby has it's own built-in email regexp:
`URI::MailTo::EMAIL_REGEXP`